pool/app/email/event.ml

```diff
@@ -195,40 +209,48 @@ let handle_event pool : event -> unit Lwt.t =
   let open Utils.Lwt_result.Infix in
   let ctx = Pool_tenant.to_ctx pool in
   function
-  | Sent email -> Service.Email.send ~ctx:(Pool_tenant.to_ctx pool) email
+  | Sent email ->
+    let%lwt sender = sender_of_pool pool in
```

Comment:
move this line directly after `let ctx ...` then you can remove them from all cases

Reply:
Some branches are not using the sender, this would save 2 DB calls in those cases.